packages/@sanity/form-builder/src/inputs/Image/ImageInput.js

```diff
@@ -240,7 +240,11 @@ export default class ImageInput extends React.PureComponent {
     return (
       <div style={{display: 'flex', flexDirection: 'row', width: 800}}>
         <div style={{width: '40%'}}>
-          <ImageTool value={{hotspot, crop}} src={imageUrl} onChange={this.handleImageToolChange} />
+          <ImageTool
+            value={{hotspot, crop}}
+            src={`${imageUrl}?w=500&fit=max`}
```

Comment:
This will only work if the image url is behind a CDN that supports these params. Can you add these in another layer perhaps?

Reply:
Agreed. We need a solution for this. Also maybe out of the scope of this pull request.